.github/workflows/examples.yaml

```diff
@@ -288,3 +288,52 @@ jobs:
             # - name: Perform CodeQL Analysis
             #   if: ${{ github.event_name == 'push' }}
             #   uses: github/codeql-action/analyze@v1
+            - name: Perform CodeQL Analysis
+              if: ${{ github.event_name == 'push' }}
+              uses: github/codeql-action/analyze@v1
+    
+    qpg6100:
+        name: QPG6100
+        env:
+            BUILD_TYPE: gn_qpg6100
+
+        runs-on: ubuntu-latest
+
+        container:
+            image: connectedhomeip/chip-build-qpg6100:0.4.8
```

Comment:
Just change this to chip-build. It should work immediately.

Reply:
Updated to 'chip-build' - closing #3129 as there's no need anymore for a specific docker container